football_dictionaries/assignment_1.py

```diff
@@ -1,2 +1,17 @@
 def players_as_dictionaries(squads_list):
-    pass
+    #iterate over input while creating generic dictionaries from set(keys_list) values
+    #fill dict(squad) with the input from player in squads_list
+    #append to list(players) 
+    #return list(players)
+    players = []
+    keys_list = ['number', 'position', 'name', 'date_of_birth', 'caps', 'club', 'country', 'club_country', 'year']
+    
+    for player in squads_list:
+            squad = {}
+            index = 0
+            squad = dict.fromkeys(set(keys_list))
+            for stat in player:
+                squad[keys_list[index]] = player[index]
+                index += 1
+            players.append(squad)
```

Comment:
@useraddmario , this works ok, but I think it's less readable. Check @VentusN solution here to see an alternative: https://github.com/rmotr-group-projects/itp-u3-c2-football-dictionaries/pull/28/files#diff-f7815caf072d06b4452fc2cb4abd3df5R4